src/essence/Ancillary/TimeControl.js

```diff
@@ -202,36 +202,34 @@ var TimeControl = {
         if (typeof layer == 'string') {
             layer = L_.layersNamed[layer]
         }
-        if (layer.time && layer.time.enabled == true) {
-            var layerTimeFormat = d3.utcFormat(layer.time.format)
-            layer.time.current = TimeControl.currentTime // keeps track of when layer was refreshed
-
-            if (layer.type == 'tile') {
-                TimeControl.setLayerWmsParams(layer)
-
-                if (L_.toggledArray[layer.name] || evenIfOff) {
-                    L_.toggleLayer(layer)
-                    L_.toggleLayer(layer)
-                }
-            } else {
-                // replace start/endtime keywords
-                var originalUrl = layer.url
-                layer.url = layer.url
-                    .replace(
-                        '{starttime}',
-                        layerTimeFormat(Date.parse(layer.time.start))
-                    )
-                    .replace(
-                        '{endtime}',
-                        layerTimeFormat(Date.parse(layer.time.end))
-                    )
-                // refresh map
-                if (L_.toggledArray[layer.name] || evenIfOff) {
-                    await Map_.refreshLayer(layer)
-                }
-                // put start/endtime keywords back
-                layer.url = originalUrl
+        var layerTimeFormat = d3.utcFormat(layer.time.format)

```

Comment:
Does this break if a layer does not have `layer.time`? It might make more sense to move `if (layer.time && layer.time.enabled == true)` around new L209 and around L217 -> L226. Thoughts?

Reply:
Thanks! I fixed this in b1b8a1f8b16e2f1bba984464290a7319649d9e51. Originally, I thought it wouldn't matter since time is generally added below but better safe than sorry:
https://github.com/NASA-AMMOS/MMGIS/blob/0a5a6348c276b560a33a5a36b110cde8bcb8c0e3/config/js/config.js#L2025-L2038.

Reply:
Oh right! Though do remember since we have quite a few MMGIS instances and older configurations around, that we can't assume things are set.